static/js/tools/statsvar_menu.tsx

```diff
@@ -28,7 +28,7 @@ class Node extends Component<NodePropType, NodeStateType> {
     this.state = {
       checked: false,
       expanded: false,
-      nodePath: props.nodePath + "," + props.title,
+      nodePath: props.nodePath + "'" + props.title,
```

Comment:
can you make this a constant? like const SEP = "'". May need to change it in the future if we have ' in the string..

Reply:
done.